### ISSUE TITLE
[#SomeChanges Suggestion] Shadow Word: Pain and Haste 

### ISSUE THREAD

**Issue by dbyena** [ClassicWoWCommunity/cata-classic-bugs#85]
TL;DR

A. Spriest balance in wrath reflects an older philosophy that hybrid DPS < pure DPS (2)
B. This is in some tension with “bring the player not the class” philosophy, also original to Wrath (6)
C. During wrath, devs considered adding haste for SW:P (1)
D. The original decision against hasted SW:P depended on (A), but also concern that it would be overpowered (4)
E. Later changes to haste snapshotting on Corruption, if applied similarly to SW:P, should ease concerns that hasted SW:P is overpowered (5)
F. Therefore, following the “bring the player, not the class” philosophy, and taking into account these later corruption changes, it makes sense to add hasted SW:P following the corruption model

My ign is Linelo and I am an avid shadow priest theory crafter, co-creator of the shadow priest sim and am a active member of the wow classic community

Wrath of the Lich King classic (WoTLKC) is right around the corner and the classic Priest/Shadow priest discords are flooding with questions regarding the viability of shadow priest in the next classic expansion. As with any classic release there is already a “pre-defined” meta and class performance expectation. The most common response to shadow priest viability in wrath is “there is no longer a need for shadow priests as their DPS is lower than pure classes and other hybrid classes such as balance druid, and debuffs are no longer unique.” Ghostcrawler put an argument together on 10/26/09 titled “Hybrid tax (2).” In this blue post, Ghostcrawler essentially argues that hybrid classes should do less damage because they can respec to fulfill a different role while “pure” classes must reroll entirely. This argument only negatively impacts the DPS spec of the hybrid class and only dilutes spec diversity. Throughout the last few years of classic the shadow priest has been either an entire meme, or a complete support class. In vanilla classic, shadow priest is a non-existent class or a hybrid healer/DPS support. The best representation of the “hybrid tax” was TBC shadow priest where their DPS was mediocre, but utility was necessary. This type of balance allows people to play the class they enjoy without being pushed off due to not being “optimal.” In WOTLKC shadow priests go through a complete identity switch from “hybrid” support class to a “hybrid” DPS class. This is where the issues arise. If a shadow priest produces less DPS than a mage or a warlock by sheer default of being a hybrid, then playing hybrid DPS is undesirable unless it’s for encounters that require a healing spec. In this classic-meta-optimal-raiding era most high-end/competitive guilds/raids won’t be seeing many hybrid DPS at all anymore. Most Hybrids will be tanks or healers, and the DPS spots will be left for the pure DPS classes. In TBCC a shadow priests raid spot was essentially a guarantee due to their powerful support ability, but in WoTLKC shadow priests are not necessary for a raid as their DPS is lower than “pure” classes and other hybrid classes that bring similar utility. This means that a shadow priest’s DPS becomes the sole distinguishing factor that determines their raid spot (other than social factors, but generally this means “respec to benefit the raid or play your “pure” DPS alt class.`` During the 2008 BlizzCon WoW Class Panel, Ghostcrawler said “you’ll want to bring the player and not the spec in the upcoming expansion.(6)”

Patch 3.3.0 PTR (10/20/09) initially had:

“Shadowform: This talent also now causes Devouring Plague, Shadow Word: Pain, and Vampiric Touch to benefit from haste. Both the period length and the duration of these spells will be reduced by haste. In addition, the mana cost has been reduced from 32% to 13% of base mana.(1)”

This same patch added

“Glyph of Quick Decay: This glyph allows for the warlock’s haste to reduce the time between periodic damage effects of Corruption.(1)”

Both are good DPS buffs for each class. Corruption for warlocks is essentially the same Shadow Word: Pain (SW:P) for shadow priests with respect to the ability to refresh throughout the fight. In the actual release of patch 3.3 shadow form talent no longer included SW:P as a DOT that benefited from haste, but Glyph of Quick Decay was kept in. The argument that “Ghostcrawler – 11/15/09” used was:

“We removed Shadow Word: Pain from scaling with haste because we thought Shadow dps was too high with all 3 dots hasted. There is a bug where you can get big SW:P dots and then keep them rolling at that magnitude forever since the spell gets constantly refreshed. It’s a nasty bug to fix.(4)”

The use of phrases “we thought’’ and “too high” seem to be referencing the ''hybrid tax” argument (however, I expect real numbers were used in this decision but were not shared). Post 3.3.5 shadow priest DPS in original WOTLK was still below most “pure” classes and the added DPS from a hasted SW:P would not push shadow priest above classes like mage/hunter/warrior/rogue/lock especially if SW:P haste snap-shotting was updated in the 3.3 hotfix (SW:P DPS per Haste Rating is ~0.13-0.17 depending on various factors). On 12/11/09 Ghostcrawler had a post on 3.3 hotfixes one of which was:

“Rolling Corruptions no longer use the initial haste value indefinitely.(5)”

This change came after the PTR when SW:P was already decided to not benefit from haste. If this change was also implemented on SW:P, then potentially shadow priest dps would not have seemed “too high.” There are two possible paths that have been historically implemented to allow SW:P to benefit from haste. First is to add SW:P back to shadow form talent with Devouring Plague / Vampiric Touch. Second is to revert Glyph of Shadow Word: Pain (like the original proposed blue post plan (“For 3.3 we are talking about introducing three new glyphs for Shadow Word: Pain, Corruption and Rejuvenation that would allow these spells to tick faster with the more haste you have.”(03)) The second approach will potentially decrease the strength overall by forcing decisions on which glyphs to use as a shadow priest. This is also similar to how corruption was implemented for affliction locks. The currently released Glyph of Shadow Word: Pain is not expected to be used as it does not scale well. 1% of base mana for a level 80 priest will be 38-39 mana per tick, so this glyph is worth just over 64 [MP5]. Ideally this glyph would get reconfigured back to the original 3.3 state where SW:P benefits from haste with the additional changes to how haste snapshots. The “pure” vs “hybrid” argument has been going on for years, and as WoW has evolved the “hybrid” DPS gap relative to “pure” DPS has been shrinking. The philosophy shift to “bring the player, and not the class” is a step in the right direction. With WoTLKC #somechanges, myself and thousands of other dedicated shadow priests would love to be able to compete at a high level playing the class/spec that they love, and a change that can achieve a similar result seen with Retribution Paladins in TBCC which was noted by developers and appreciated by players would be a change to Shadow Word: Pain and its interactions with haste

Citations

1) “Patch 3.3 Guide: Patch Notes.” Wowhead, h*ttps://www.wowhead.com/patchnotes=3.3.0.
2) Tracker, Blue. “Blue Tracker.” World of Warcraft Blue Posts, https://www.bluetracker.gg/wow/topic/us-en/20143783047-dots-hots-and-haste/.
3) Tracker, Blue. “Blue Tracker.” World of Warcraft Blue Posts, https://www.bluetracker.gg/wow/topic/us-en/20677330431-hybrid-tax/.
4) Tracker, Blue. “Blue Tracker.” World of Warcraft Blue Posts, https://www.bluetracker.gg/wow/topic/us-en/21048545285-swp-and-haste-in-33/.
5) Tracker, Blue. “Blue Tracker.” World of Warcraft Blue Posts, https://www.bluetracker.gg/wow/topic/us-en/21730605184-impending-33-hotfixes/.
6) https://www.tentonhammer.com/articles/blizzcon-2008-wow-classes-panel

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
